graphql/ui_plugin_build_baron.go

```diff
@@ -115,21 +115,35 @@ func BbGetCreatedTicketsPointers(taskId string) ([]*thirdparty.JiraTicket, error
 	return results, nil
 }
 
-func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, bool, error) {
-	projectNotFoundError := false
+type buildBaronConfig struct {
+	ProjectFound     bool
+	SearchConfigured bool
+}
+
+func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, buildBaronConfig, error) {
+	bbConfig := &buildBaronConfig{}
```

Comment:
what's the benefit of initializing a pointer instead of a `buildBaronConfig{}`?

Reply:
I don't think there is. I'll change it.